Dev

Niketanjha opened the pull request.

Niketanjha committed:
second commit

Niketanjha committed:
withbuildfile

Niketanjha committed:
second commit

Niketanjha committed:
second commit

ravi1225 commented on src/App.js:
```diff
@@ -1,25 +1,65 @@
-      </header>
-    </div>
-  );
+class App extends React.Component{
```
use functional  component

Niketanjha committed:
build file  ignored

Niketanjha committed:
minor fixes

Niketanjha committed:
2. make button component
3. Sample heading compnent   //done
4. fixed card width
5. fixed card font-size
6.fixed card font-color
